docs/use.md

```diff
@@ -327,6 +328,18 @@ function Calculator() {
 }
 ```
 
+### Call a function on render
+
+If you want to call a function when a component renders, instead of in response to user interaction, you can use the `useCallableFunctionResponse` hook.
+
+```jsx
+function LikeCount({ videoId }) {
+  const { status, data: likeCount } = useCallableFunctionResponse('countVideoLikes', { data: { videoId: videoId } });
```

Comment:
What do you think about the name `useCallableFunctionResponse`? It's a little wordy because I wanted to emphasize that this would call the callable function and update with the response. Any suggestions to make it shorter/better?

Reply:
Wordy but I like that it's very explicit. SGTM